opm/autodiff/BlackoilPropsAdFromDeck.hpp

```diff
@@ -59,6 +59,24 @@ namespace Opm
     {
         friend class BlackoilPropsDataHandle;
     public:
+        typedef typename SaturationPropsFromDeck::MaterialLawManager MaterialLawManager;
+
+        /// Constructor wrapping an opm-core black oil interface.
```

Comment:
This comment is wrong -- it has also been wrong for a while for the existing constructors. I would appreciate it if you could correct this, and at the same time give a (short) explanation in the docs here for why the new constructors are provided (not recreating the material law manager).